document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -34,7 +34,7 @@
 
     @Override
     public DocumentInfoResponse getDocumentInfo(DocumentInfoRequest documentInfoRequest) throws DocumentInfoException {
-        LOG.info("Started getting document");
+        LOG.info("Started getting document info");
```

Comment:
Other APIs are using an interceptor to log start and end requests (outputs the time taken for the request to complete which helps if there's a performance issue - see company accounts API for an example). This log will be output every time a document is requested and an accounts submission but it has no details in it to link it to which document is being generated.